examples/eth_bots/config.py

```diff
@@ -0,0 +1,72 @@
+"""Experiment configuration"""
```

Comment:
this file is not supposed to be here, probably accidental rebase issue. delete it